package.json

```diff
@@ -4,7 +4,6 @@
   "description": "CDS plugin providing out-of-the box support for automatic capturing, storing, and viewing of the change records of modeled entities.",
   "repository": "cap-js/change-tracking",
   "author": "SAP SE (https://www.sap.com)",
-  "homepage": "https://cap.cloud.sap/",
```

Comment:
why remove?

Reply:
I also think this should stay as is for now. 

@danjoa: Let's keep it in and we can later on directly link to the envisioned CDS plugin landing page?

Reply:
I think default is github readme which is better than cap.cloud.sap in this case, as it is a plugin.